### ISSUE TITLE
Extensions view:Screen reader does not announces the selected/not selected state "Details, Extension Pack, Contribution changelog" buttons.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/vscode#88758]
**Environment Details:​**
VSCode Version: 1.42.0-insider (System setup)
OS Version: Win10

**Steps to Reproduce:**
1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Online"
3. Navigate  right panel till "Extension: Visual Studio Online" 
4. Start the Narrator/NVDA.
5. Navigate to " Details, Extension Pack, Contribution changelog" buttons.
6. Observe screen reader selected state or not.

**Actual:**
Screen reader does not announces the selected/not selected state for "Details, Extension Pack, Contribution changelog" buttons.

**Expected:​**
Screen reader should announces the selected/not selected state for "Details, Extension Pack, Contribution changelog" buttons.

**​User Impact:​**
Screen reader user does not understand which control is selected/not selected if it does not announces.

**Comment by isidorn** [comment]
This feedback makes sense. The extensions details view needs some accessibility love. Assigning for now to @sandy081 and he can forward to whoever owns it

**Comment by isidorn** [comment]
Closing in favor of https://github.com/microsoft/vscode/issues/88792
